e2e/tests/stablex_test.rs

```diff
@@ -70,16 +70,18 @@ fn test_with_ganache() {
         .from(Account::Local(accounts[1], None))
         .wait_and_expect("Cannot place first order");
 
-    let batch_id = instance.get_current_batch_id().call().wait().unwrap();
-    close_auction(&web3, &instance);
-    let wait_time = Duration::from_secs(310);
+    let batch_id = instance
+        .get_current_batch_id()
+        .wait_and_expect("Failed to query batch ID");
+    let wait_time = Duration::from_secs(340);
```

Comment:
Is the change from `310` to `340` especially related to this PR?

Reply:
It is related although could have arguably been its own PR. Specifically, this PR was for fixing intermittent Ganache e2e test failures and the incorrect duration was causing errors. The main issue was advancing EVM time, but the incorrect duration was a minor secondary issue.